src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/AvoidInlineConditionalsCheck.java

```diff
@@ -42,6 +42,23 @@
  * <pre>
  * &lt;module name=&quot;AvoidInlineConditionals&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * int x = 5;
+ * boolean foobar = (x == 5); // OK
+ *
+ * String text;
+ * text = (text == null) ? "" : text; // violation
+ *
+ * String b;
+ * if (a != null &amp;&amp; a.length() &gt;= 1) { // OK
+ *   b = a.substring(1);
+ * } else {
+ *   b = null;
+ * }
+ *
+ * b = (a != null &amp;&amp; a.length() &gt;= 1) ? a.substring(1) : null; // violation
```

Comment:
is there other cases rather than ternary operator?
They should be shown as well

Reply:
No, this check only finds `?` in the code, and reports violation if found.

Reply:
Ok. Check has a weird (for me) naming then.